doc/manual/manual.tex

```diff
@@ -2045,18 +2053,26 @@ \subsection{Installing and running \aspect{} on Mac OS X}
 \end{verbatim}}
 \dealii{} app opens a terminal window and displays a \dealii{} message.   \dealii{}.app will install all required libraries for \aspect{} (p4est, parMeTiS, and Trilinos) and will include the environment variables needed to use these libraries.
 
+To ensure the correct compilers are picked up when configuring \aspect{}, add the following lines to your .profile or .bash\_profile:
```

Comment:
mark up these file names with `\texttt{...}`. Maybe qualify these names with `~/` to indicate that they're in the home dir?